Content.Server/MobState/Components/DeleteOnDeathComponent.cs

```diff
@@ -0,0 +1,9 @@
+namespace Content.Server.MobState.Components
+{
+    /// <summary>
+    /// For shrimple mobs that do not leave a corpse
+    /// </summary>
+    [RegisterComponent]
+    public sealed class DeleteOnDeathComponent : Component
+    {}
```

Comment:
shouldnt this just be handled by destructible?

Reply:
yeah I can't remember why I originally did it but it works now. a bunch of stuff has changed since then